changelog/14039.expired.rst

```diff
@@ -0,0 +1,4 @@
+Deprecated ``np.rank`` removed
+------------------------------
+
+ ``np.rank`` has been removed. This was deprecated in numpy 1.10 and has been replaced by ``np.ndim``.
```

Comment:
Somewhat think last time around we just did a bullet point list here. Not sure it matters, I think I will just change it later (and then hope we actually do it like that) ;). Thanks!

Reply:
Sure. Thanks!